product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+* Public IP Addresses – Standard: number of clusters you plan to deploy in a given region
+
+### vCPU quota
+
+Any time a new VM is deployed the vCPUs for the VMs must not exceed the vCPU quota for the VM size family or the total regional vCPU quota. The VMs deployment fails if either of those quotas are exceeded. 
+
+!!! Note
+    The default number of total cores per subscription per region is 20. See [Virtual Machines limits - Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#virtual-machines-limits---azure-resource-manager) for more information.
+
+In each region, EDB Cloud uses six ESv3 and six DSv2 virtual machine cores to manage your EDB Cloud infrastructure.
+Your clusters deployed in the region use separate ESv3 virtual machine cores. The number of cores depends on the Instance Type and High Availability options of the clusters you provision. You can calculate the number of ESv3 cores required for your cluster based on the following:
+
+* Virtual machine instance of type E{N}Sv3 uses {N} cores. For example, an instance of type E64Sv3 uses 64 ESv3 cores.
+* Cluster running on an E{N}Sv3 instance with high availability not enabled uses exactly {N} ESv3 cores.
+* Cluster running on an E{N}Sv3 instance with high availability enabled uses 3 * {N} ESv3 cores.
+
+As an example, if you provision the largest virtual machine E64Sv3 with high availability enabled, it requires (3 * 64)=192 ESv3 cores per region. EDB Cloud infrastructure requires an additional six ESv3 and six DSv2 virtual machine cores per region.
+
+**Recommended quota**
+
+EDB recommends the following when requesting Virtual Machine resource quota increases:
+* Total Regional vCPUs: minimum of 100+ per designated region
+* Standard ESv3 Family vCPUs: minimum of 50+ per designated region
+* Standard DSv2 Family vCPUs: minimum of 20+ per designated region
+
+### Virtual Machine (SKU) quota
+
+When you deploy your clusters in HA mode, EDB Cloud uses multiple availability zones in the region where you plan to deploy the cluster. EDB Cloud uses Standard DSv2 Family vCPUs to manage workloads and Standard ESv3 Family vCPUs to manage clusters.
+
+**Recommended quota**
+
+Ensure that all regions and zones designated for the deployment of your clusters have no SKU restrictions for the vCPU families used by EDB Cloud. `Standard_DS2_v2` and `Standard_E2s_v3` in zone 1,2,3 of every region designated for cluster deployment should have no restrictions.
+
+### Azure Service Provider registrations
+
+EDB Cloud creates and manages [certain types of Azure resources](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types). Ensure that the following Azure service providers have been registered in your Azure subscription. Absence of registration for any of these service providers in your Azure subscription leads to failure to create clusters in EDB Cloud.
```

Comment:
> Ensure that the following Azure service providers have been registered in your Azure subscription

Again, this is handled in the page that describes the preflight script, yes? We're instructing customers to do a lot of things on this page that we should be handling automatically via the script.

Reply:
In the new structure, we recommend all users (those who are using the script and those using the manual steps) to read through the understanding page. 

I have let this section be, as is, for the users not using the script.